dlk/python/dlk/templates/Makefile.tpl

```diff
@@ -138,8 +148,8 @@ clean:
 lm_x86:           CXX = g++
 lm_x86:           FLAGS += $(INCLUDES) -O3 -std=c++0x -g -DUSE_PNG -pthread -g
 
-lm_aarch64:       CXX = aarch64-linux-gnu-g++
-lm_aarch64:       FLAGS += $(INCLUDES) -O3 -std=c++0x -g -DUSE_NEON -DUSE_PNG -pthread -g
+lm_aarch64:           CXX = aarch64-linux-gnu-g++
+lm_aarch64:           FLAGS += $(INCLUDES) -std=c++0x -O3 -DUSE_ASIMD -DUSE_PNG -pthread -g -fopenmp
```

Comment:
could we change indentation to the original one?

Reply:
OK